src/components/ui/date-field.tsx

```diff
@@ -101,7 +101,7 @@ export default function DateField({
   };
 
   return (
-    <div className="flex items-center gap-2">
+    <div className="flex flex-wrap items-center gap-4 md:gap-2 ">
```

Comment:
this doesn't look good. why not keep it in one line? day and month is two digits, we could reduce it's width.

<img width="461" alt="image" src="https://github.com/user-attachments/assets/4d486936-cca6-4064-b853-26d97905735b" />